src/site/markdown/server/manage-cli.md

```diff
@@ -0,0 +1,264 @@
+Manage James via the Command Line
+=================================
+
+With any distribution, James is packed with a command line client.
+
+To use it enter, for Spring distrubution:
+
+```
+./bin/james-cli.sh -h 127.0.0.1 -p 9999 COMMAND
+```
+
+And for Guice distributions:
+
+```
+java -jar /root/james-cli.jar -h 127.0.0.1 -p 9999 COMMAND
+```
+
+The following document will explain you which are the available options for **COMMAND**.
+
+Note: the command line before **COMMAND** will be documente as *{cli}*.
+
+## Manage Domains
+
+Domains represent the domain names handled by your server.
+
+You can add a domain:
+
+```
+{cli} AddDomain domain.tld
+```
+
+You can remove a domain:
```

Comment:
The domain is not here, but I don't think there is other consequences. (User belonging to the domain are not removed).